src/components/Popover/Anchor.tsx

```diff
@@ -0,0 +1,10 @@
+import {
+  Anchor as RadixPopoverAnchor,
+  PopoverAnchorProps as RadixPopoverAnchorProps,
+} from "@radix-ui/react-popover";
+
+export type PopoverAnchorProps = RadixPopoverAnchorProps;
+
+export const Anchor = ({ children, ...props }: PopoverAnchorProps) => {
+  return <RadixPopoverAnchor {...props}>{children}</RadixPopoverAnchor>;
+};
```

Comment:
suggestion: imo there is no sense to create such a wrapper, why not use RadixPopoverAnchor direct in Arrow?